src/py/flwr/client/client_app.py

```diff
@@ -30,23 +30,47 @@
 from .typing import ClientAppCallable
 
 
+def _alert_erroneous_client_fn() -> None:
+    raise ValueError(
+        "A `ClientApp` cannot make use of a `client_fn` that does "
+        "not have a signature in the form: `def client_fn(context: "
+        "Context)`. You can import the `Context` type from `flwr.common`."
```

Comment:
Let's expand that to make it more obvious for users. How about:
```suggestion
        "Context)`. You can import the `Context` like this:"
        "    `from flwr.common import Context`"
```